### PR TITLE
Add security disclaimer and contact info

### PR DESCRIPTION
**Pull request by eloylp**
**Summary of changes**

After internal conversations, we have:

* A security related affairs email for receiving sec issues privately.
* Ensure we are clear about the current status of the project by updating disclaimers.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/security.md [hunk 1]
```diff
@@ -7,4 +7,7 @@ assignees: ''
 
 ---
 
-Please for opening security issue, please contact tbd@eiger.co . For security reasons, DO NOT PUBLISH HERE INFORMATION.
+⚠️ Disclaimer: The current contracts are POC and are still not
+reviewed by security teams. We do not encourage their use in production yet.
+
+Please for opening security issue, please contact hello@eiger.co . For security reasons, DO NOT PUBLISH HERE INFORMATION.
```
Comment (mariopil) [inline .github/ISSUE_TEMPLATE/security.md:13]: Double `please`. I would rephrase it to:
`For opening security issue, please contact hello@eiger.co `

**Review by mariopil — APPROVED** [review]

**Review by geofmureithi — APPROVED** [review]
Apart from the double please. LGTM
